### ISSUE TITLE
lollms not working in a few ways.

### ISSUE THREAD

**Issue by photogbill** [ParisNeo/lollms_legacy#5]
The personality maker throws this error:

Received message : batman - yes i tried to make batman.  
Started generation task
Exception in thread Thread-38 (start_message_generation):
Traceback (most recent call last):
  File "D:\lollm\lollms-webui\installer_files\lollms_env\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "D:\lollm\lollms-webui\installer_files\lollms_env\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "D:\lollm\lollms-webui\lollms-webui\api\__init__.py", line 1287, in start_message_generation
    self.discussion_messages, self.current_message, tokens = self.prepare_query(client_id, message_id, is_continue)
  File "D:\lollm\lollms-webui\lollms-webui\api\__init__.py", line 1001, in prepare_query
    return discussion_messages, message.content, tokens
UnboundLocalError: local variable 'tokens' referenced before assignment

ok, lets try micky mouse talking.. 

Mounted Mickey Mouse talking to kids
Client FW5VQXcFshNxmetRAAAB requested cancelling generation
Thread not alive
Client FW5VQXcFshNxmetRAAAB canceled generation
Starting message generation by Mickey Mouse talking to kids
Text generation requested by client: FW5VQXcFshNxmetRAAAB
Received message : hi
Started generation task
Exception in thread Thread-91 (start_message_generation):
Traceback (most recent call last):
  File "D:\lollm\lollms-webui\installer_files\lollms_env\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "D:\lollm\lollms-webui\installer_files\lollms_env\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "D:\lollm\lollms-webui\lollms-webui\api\__init__.py", line 1287, in start_message_generation
    self.discussion_messages, self.current_message, tokens = self.prepare_query(client_id, message_id, is_continue)
  File "D:\lollm\lollms-webui\lollms-webui\api\__init__.py", line 1001, in prepare_query
    return discussion_messages, message.content, tokens
UnboundLocalError: local variable 'tokens' referenced before assignment

Basically its not working.  Its weird, not sure what's causing this.  Worked fine the other day
